Update FAQ items (#115)
ScratchAddons/website-v2: content/docs/faq.md

### Is this the same as TurboWarp Addons?

[TurboWarp](https://turbowarp.org/) has some addons from Scratch Addons which can be used on their editor without installing Scratch Addons. However, Scratch Addons also includes addons for the Scratch website and the popup display. Therefore, it is still useful to have Scratch Addons even if you only code using TurboWarp.

## System Requirements

### What are the system requirements for Scratch Addons?

Scratch Addons is officially supported on [Google Chrome](https://google.com/chrome/) (version 80 and above) and [Mozilla Firefox](https://mozilla.org/firefox/) (version 74 and above). Scratch Addons should also work on other desktop browsers that are based on those browsers, such as:

- [Microsoft Edge](https://microsoft.com/edge/) (version 80+)
- [Opera](https://opera.com/download/) (version 67+)
- [Vivaldi](https://vivaldi.com/download/) (version 2.11+)
- [Brave](https://brave.com/download/) (version 1.3+)

> Additionally, Scratch Addons (for obvious reasons) requires Internet connection.

Reviewer: TW desktop go brrr

Reviewer 2: Yeah, it would be great to mention turbowarp desktop.

### Can I use Scratch Addons on a mobile device?

> For Android users: Yes, but it is not recommended. Major browsers do not allow Scratch Addons (or any other extensions) to be installed, so you need to use browsers such as [Kiwi](https://kiwibrowser.com/) to do so. Scratch Addons' UI is not well-tested on touchscreens or environments with small viewport size, so some of the features might not work as expected.

Reviewer: `screen` instead of `viewport`

### Can I use Scratch Addons on the offline editor?

Most of the project editor addons are also available on [TurboWarp](https://turbowarp.org/) which has a [downloadable app](https://desktop.turbowarp.org/) for Windows, macOS, and Linux. Additionally, on browsers that support Progressive Web Application (PWA) such as Google Chrome on Chromebook, you can also install the TurboWarp editor as a PWA and use it without Internet connection.

Scratch Addons cannot be used on the official Scratch app.

### Are there any incompatible programs?

Some browser extensions and userscripts may interfere with Scratch Addons. If you experienced problems, you should turn off those programs:

- Scratch 3 Developer Tools: This browser extension is a copy of the Developer tools addon. You should uninstall the Developer Tools browser extension and turn on the addon instead.
- Better3.0: This browser extension can interfere with some addons. Luckily, most of its features are also available as addons.

> - Redux DevTools: This can interfere with the internal workings of Scratch Addons. Disable it if you are not using.

Reviewer: The last sentence of this line sounds off

Reviewer 2: I'll fix it

- Scratch 3 Developer Tools: This browser extension is a copy of the Developer tools addon. You should uninstall the Developer Tools browser extension and turn on the addon instead.
- Better3.0: This browser extension can interfere with some addons. Luckily, most of its features are also available as addons.

> - Redux DevTools: This can interfere with the internal workings of Scratch Addons. You should disable Redux DevTools extension if you are not using it.

Reviewer: ```suggestion
- Redux DevTools: This can interfere with the internal workings of Scratch Addons. You should disable it if you are not using it.
```

### Will my account be safe when using Scratch Addons?

> Scratch Addons doesn't use your account credentials to essentially work. In fact, you can be logged out from Scratch, and Scratch Addons will still work. Scratch Addons will only send requests based on the cookies that you have, which is supplied by the browser for each request, so some addons like Scratch Messaging won't work when you are logged out, but it won't affect other parts of the extension.

Reviewer: ```suggestion
Scratch Addons doesn't use your account credentials to essentially work. In fact, you can be logged out from Scratch, and Scratch Addons will still work. Scratch Addons will only send requests based on the cookies that you have, which are supplied by the browser for each request. So, some addons like Scratch Messaging won't work when you are logged out, but the majority of the extensions features should not be affected.
```

- **Contribute some code**
  

>   If you can code on JavaScript, HTML, and CSS, you can contribute by doing some coding/programming. You can fix bugs, tackle some requests, or create your own addon.

Reviewer: coding on javascript
